### ISSUE TITLE
[SW-Content]  Copyedit and publish Final Content QA for 5103 evidence page [CE]

### ISSUE THREAD

**Issue by SgardnerPMP** [department-of-veterans-affairs/va.gov-team#37650]
## Story
Support copy editing of original story on #2763

## Resources
#27635
Beth Pots

## TASKS
- [x] Review and copy edit work completed on 5103/evidence page for R&S
- [x] Complete prepub checklist
- [x] Confirm with Product Owner for last tweaks (Beth)

## Acceptance Criteria
- [x] Product Owner has reviewed
- [x] Publish page

**Comment by RLHecht** [comment]
I did a final copyedit and completed the pre-publishing checklist. Just waiting on final stakeholder approval to publish the page.

@psanders86 These are what I'm suggesting for the pages to add this link to. Let me know what you think:

https://www.va.gov/pension/how-to-apply/ At end of bullets in featured content block. [Learn about the evidence you'll need to support your claim]

https://www.va.gov/pension/aid-attendance-housebound/ At end of bullet points in How do I get this benefit section?  [Learn about the evidence you'll need to support your claim]

https://www.va.gov/pension/survivors-pension/ At end of How do I apply for a VA Survivors Pension section [Learn about the evidence you'll need to support your claim]

https://www.va.gov/disability/dependency-indemnity-compensation/ At the end of each accordion for Am I eligible for VA DIC as a surviving spouse or dependent? [Learn more about the evidence you'll need to support your claim]

As a related link on https://www.va.gov/resources/how-are-pension-benefits-and-disability-compensation-different/

**Comment by psanders86** [comment]
Yes, I think those locations make sense. Thank you @RLHecht

**Comment by SgardnerPMP** [comment]
@RLHecht to Proof and should be ready to publish with approval.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Would you take a look at the placement changes I just made? I moved up the "documents you'll need" section within the first 3 H2s. Once you're done, pause and I'll email stakeholders for their final final final review.

**Comment by RLHecht** [comment]
@bethpottsVADEPO I reviewed your changes and all looks good to me.

**Comment by RLHecht** [comment]
I published this page and added the article plus 2 others to the R&S landing page. @bethpottsVADEPO Let me know if I should add related links to this on the pages I listed a few comments above. Moving this to validate.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Go ahead and add the crosslinking as you listed above. I think we'll need to standardize these pages as they all take a different approach, but for now, this looks good. Thanks!

**Comment by lunascoop** [comment]
@RLHecht these crosslinks are ready for review.

How to apply for a VA pension as a Veteran: http://preview-prod.vfs.va.gov/preview?nodeId=586
VA Aid and Attendance benefits and Housebound allowance: http://preview-prod.vfs.va.gov/preview?nodeId=584
VA Survivors Pension: http://preview-prod.vfs.va.gov/preview?nodeId=590
About VA DIC for spouses, dependents, and parents: http://preview-prod.vfs.va.gov/preview?nodeId=855
Fully developed claim for a VA pension: http://preview-prod.vfs.va.gov/preview?nodeId=587
How are pension benefits and disability compensation different?: http://preview-prod.vfs.va.gov/preview?nodeId=36106

**Comment by RLHecht** [comment]
Thanks, @lunascoop I've published all of these.

**Comment by bethpottsVADEPO** [comment]
@RLHecht Can we close this ticket now?
@lunascoop Wasn't there another ticket for deleting the "trial" pages you had mocked up for me (except the accrued benefits ones) when we thought we'd have to create 10-12 new pages? Is that already done? I couldn't find that ticket just now, so thought I'd ask in the context of this other 5103 ticket. You can respond in slack if you want!

**Comment by lunascoop** [comment]
@bethpottsVADEPO I think you're referring to [ticket #27635](https://github.com/department-of-veterans-affairs/va.gov-team/issues/27635). This slipped my radar but I can delete the trail pages this sprint (except for the separate accrued benefit pages)
